packngo.go

```diff
@@ -148,6 +152,10 @@ func (c *Client) Do(req *http.Request, v interface{}) (*Response, error) {
 
 	response := Response{Response: resp}
 	response.populateRate()
+	if os.Getenv(debugEnvVar) != "" {
```

Comment:
this does a syscall for each req/response, I'd rather have one call done in `NewClient` and cache it

Reply:
OK. I've just added it.